e2e_test/shared.sh

```diff
@@ -18,15 +18,15 @@ case $NETWORK in
       echo "Using mainnet network"
```

Comment:
Do we want to change this for Mainnet as well?

Reply:
Yes! I missed it